### DISCUSSION TITLE
如何在最终插件里去掉 ts 文件

### DISCUSSION THREAD

**Discussion by linonetwo** [tiddly-gittly/Modern.TiddlyDev#43] [Q&A]
![图片](https://github.com/tiddly-gittly/Modern.TiddlyDev/assets/3746270/035c004f-eab7-4b8a-987e-4b92e097ae52)

可以看到，

[E:\repo\watch-fs\src\watch-fs\deep-equal.ts](http://127.0.0.1:8080/#E%3A%5Crepo%5Cwatch-fs%5Csrc%5Cwatch-fs%5Cdeep-equal.ts)
[E:\repo\watch-fs\src\watch-fs\inverseFileIndex.ts](http://127.0.0.1:8080/#E%3A%5Crepo%5Cwatch-fs%5Csrc%5Cwatch-fs%5CinverseFileIndex.ts)
[E:\repo\watch-fs\src\watch-fs\utils.ts](http://127.0.0.1:8080/#E%3A%5Crepo%5Cwatch-fs%5Csrc%5Cwatch-fs%5Cutils.ts)

这三个文件应该不是必须的

**Comment by Gk0Wk** [comment]
好像只有windows上会有这个问题

**Comment by Gk0Wk** [comment]
和Issue #35 问题相同，已修复
